.gitignore

```diff
@@ -5,5 +5,6 @@ models/**
 db/
 static/
 .idea/
+embeding_models/e5-large-v2/
```

Comment:
**issue:** this shouldn't be like that, it's a submodule

Reply:
I remove this and add new .gitignore